index.html

```diff
@@ -18,8 +18,31 @@ <h1>Daniel Eicher</h1>
       </ul>
     </nav>
   </header>
+  <figure>
+    <img src="https://source.unsplash.com/random/600x300" alt="Beispielbild">
+    <figcaption>Dies ist ein Beispielbild</figcaption>
+  </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <li><b>Phone: </b>+34 642 104 722</li>
+      <li><b>Mail: </b>danieleicher@gmx.net</li>
```

Comment:
good idea to improve the contrast here , i am curious whether we are taught to do this the way you did or take CSS